### PR TITLE
Fixes: #16083 - Add font-variant-ligatures setting to disable ligatur…

### LINKED ISSUE #16083: Weird font style when doing name that contains "<number>x<number>"
### Deployment Type

Self-hosted

### NetBox Version

v4.0.1

### Python Version

3.10

### Steps to Reproduce

1. Make a device (or anything really) and put something similar to 0x3 or 0x2 in the name.

### Expected Behavior

A normal looking x in the name

### Observed Behavior

A weird style of font on the x.
image
image

### PR DESCRIPTION
**Pull request by DanSheps**
### Fixes: #16083 - Add font-variant-ligatures setting to disable ligatures on chromium

* Add font-variant-ligatures setting and set to none in _tabler.scss overrides file.
* Bundle styles

### CODE REVIEW

**Review by jeremystretch — CHANGES_REQUESTED** [review]

File: netbox/project-static/styles/overrides/_tabler.scss [hunk 1]
```diff
@@ -1,4 +1,8 @@
 // Restore default foreground & background colors for <pre> blocks
+body {
```
[the remark this answers is not in the document]
Comment (DanSheps) [reply netbox/project-static/styles/overrides/_tabler.scss:3]: Fixed it up, sorry about that

**Comment by jeremystretch** [comment]
Seems like it's still happening inside input fields. @DanSheps could you take a look? Seems fine when rendered as static text on the page. (This is on Firefox btw.)

screenshot

**Comment by DanSheps** [comment]
> Seems like it's still happening inside input fields. @DanSheps could you take a look? Seems fine when rendered as static text on the page. (This is on Firefox btw.)

Looks like chrome has a specific override for `<input>` within the browser CSS.  Likely the same in Firefox.  I added `<input>` to the override file

**Comment by jeremystretch** [comment]
There are other elements (e.g. `<textarea>`) where this also needs to be applied, so I simplified it to a global rule.

**Review by jeremystretch — APPROVED** [review]
Thanks @DanSheps!

**Pull request merged**